statics/js/components/graph-layout.js

```diff
@@ -125,7 +125,7 @@ var LinkLabelLatency = Vue.extend({
     },
 
     flowCategoryKey: function(flow) {
-      return `a=${flow.ANodeTID} b=${flow.BNodeTID} app=${flow.Application}`;
+      return `app=${flow.Application}`;
```

Comment:
I think here Application is not enough, You could use link.A/B or network.A/B

Reply:
done. added link.A